cve_bin_tool/checkers/python.py

```diff
@@ -62,12 +62,7 @@ def get_version(lines, filename):
     VPkg: python, python
     """
     version_info = dict()
-    if (
-        filename[::-1].startswith(("python")[::-1])
-        or filename[::-1].startswith(("python3.6")[::-1])
-        or filename[::-1].startswith(("python3.7")[::-1])
-        or filename[::-1].startswith(("python3.8")[::-1])
-    ):
+    if "python" in filename or "Python" in filename or "PYTHON" in filename:
```

Comment:
This can be simplified as 
`if "python" in filename.lower():`

I'm not 100% sure that behaves exactly as expected for all languages and character sets, but it would at least cover these three cases ok.